public-ui/src/app/data-browser/views/survey-chart/survey-chart.component.html

```diff
@@ -19,6 +19,7 @@
 <div class="chart" *ngIf="!displayGraphErrorMessage">
   <app-biosex-chart-react *ngIf="isBioSexChart(); else angChart" (click)="$event.stopPropagation()" [domain]="'survey'" [genderAnalysis]="selectedChartAnalysis" [genderCountAnalysis]="surveyCountAnalysis.genderCountAnalysis" [selectedResult]="selectedResult"></app-biosex-chart-react>
   <app-age-chart-react *ngIf="isAgeChart(); else angChart" (click)="$event.stopPropagation()" [domain]="'survey'" [ageAnalysis]="selectedChartAnalysis" [ageCountAnalysis]="surveyCountAnalysis.ageCountAnalysis" [selectedResult]="selectedResult"></app-age-chart-react>
+  <app-version-chart-react *ngIf="isVersionChart(); else angChart" (click)="$event.stopPropagation()" [versionAnalysis]="selectedChartAnalysis" [surveyVersionAnalysis]="versionAnalysis" [selectedResult]="selectedResult"></app-version-chart-react>
```

Comment:
The conditions here need to be modified so it only renders one chart at a time. Currently it's rendering each chart three times (one React and two Angular) since each `*ngIf` has the `else angChart` option. Also noticed the months on the Angular version charts are out of order, not sure how critical that is:
![localhost_4201_survey_covid-19-participant-experience](https://user-images.githubusercontent.com/40036095/112743485-5ffbcc80-8f5d-11eb-857a-7c9865fc7519.png)